apps/wing/src/commands/compile.ts

```diff
@@ -48,6 +48,13 @@ export interface ICompileOptions {
   readonly target: Target;
 }
 
+/**
+ * Normalizes paths from windows to posix.
+ */
+function normalPath(path: string) {
```

Comment:
ℹ️  I hesitated on adding a utils.ts or something, but I can do that if this is ugly

Reply:
Since you're calling that `.replace(...)` from multiple places and since almost every app or lib on our monorepo has some sort of a utils module, I would add utils.ts, but I'm leaving it to you to decide and approving your PR anyway.

Reply:
Alright you convinced me